### PR TITLE
Updates to Snowpark DE for Codespaces

### PR DESCRIPTION
**Pull request by sfc-gh-jhansen**
This PR updates the Data Engineering Pipelines with Snowpark Python Quickstart to use Codespaces!

### CODE REVIEW

File: site/sfguides/src/data_engineering_pipelines_with_snowpark_python/data_engineering_pipelines_with_snowpark_python.md [hunk 1]
```diff
@@ -57,22 +57,11 @@ You will learn about the following Snowflake features during this Quickstart:
 ### What You’ll Need
 You will need the following things before beginning:
 
-* Snowflake
+* Snowflake account
     * **A Snowflake Account**
     * **A Snowflake user created with ACCOUNTADMIN permissions**. This user will be used to get things setup in Snowflake.
     * **Anaconda Terms & Conditions accepted**. See Getting Started section in [Third-Party Packages](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-packages.html#getting-started).
-* Miniconda
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/data_engineering_pipelines_with_snowpark_python/data_engineering_pipelines_with_snowpark_python.md:64]: How cool does it feel to remove this portion! 😎

**Review by sfc-gh-ghernandez — APPROVED** [review]
I'm approving – also wondering before we merge though: did we need to update any of the commands that are run from the command line? (i.e., `snow snowpark function create` vs `snow function create`, etc. @sfc-gh-jhansen

File: site/sfguides/src/data_engineering_pipelines_with_snowpark_python/data_engineering_pipelines_with_snowpark_python.md (hunk 1 above)
Comment (sfc-gh-jhansen) [reply site/sfguides/src/data_engineering_pipelines_with_snowpark_python/data_engineering_pipelines_with_snowpark_python.md:64]: So good :)

**Comment by sfc-gh-jhansen** [comment]
Thanks @sfc-gh-ghernandez! I wish we could update this Quickstart now to use the new version of SnowCLI but it is not ready. It still does not support project configurations which are very important overall, and critical for the final CI/CD step. So for now I have it pinned to the older version. I'm working with the product team and pushing to get it fixed. Once it's fixed I will definitely be updating this Quickstart again to use the new version.

**Pull request merged**